app/models/manageiq/providers/ansible_tower/shared/inventory/persister/automation_manager.rb

```diff
@@ -9,6 +9,6 @@ module ManageIQ::Providers::AnsibleTower::Shared::Inventory::Persister::Automati
     has_automation_manager_configuration_script_payloads :model_class => ManageIQ::Providers::Inflector.provider_module(self)::AutomationManager::Playbook
     has_automation_manager_configured_systems
     has_automation_manager_inventory_root_groups
-    has_vms :parent => nil, :arel => Vm, :strategy => :local_db_find_references
+    has_vms :name => :vms, :parent => nil, :arel => Vm, :strategy => :local_db_find_references
```

Comment:
Too late to the party 😄  but I think `:name => :vms` should probably be in the `has_vms` method definition.